_src/static/sass/base/_base.scss

```diff
@@ -0,0 +1,92 @@
+/* ============================
+-------------------------------
+ 1. Basic Setup
+-------------------------------
+============================ */
+
+
+/* ============================
+1. Basic Setup
+---------------------------- */
+html {
+    height: 102%;
```

Comment:
Yeah, I'd say let's either set it to 100% or remove it for now. I believe that scroll issue was a Studio bug. If it comes up with us, we'll whack it.